packages/ses/src/error/tame-error-constructor.js

```diff
@@ -32,29 +32,29 @@ export default function tameErrorConstructor(
   stackFiltering = 'concise',
 ) {
   if (errorTaming !== 'safe' && errorTaming !== 'unsafe') {
-    throw new Error(`unrecognized errorTaming ${errorTaming}`);
+    throw new FERAL_ERROR(`unrecognized errorTaming ${errorTaming}`);
```

Comment:
These throws could still be TypeErrors, just because that would be more expected anyway.